drivers/net/ethernet/renesas/rswitch.c

```diff
@@ -1439,6 +1437,31 @@ int rswitch_tx_free(struct net_device *ndev, bool free_txed_only)
 	return free_num;
 }
 
+int rswitch_tx_poll(struct napi_struct *napi, int budget)
+{
+	struct net_device *ndev = napi->dev;
+	struct rswitch_device *rdev = ndev_to_rdev(ndev);
+	struct rswitch_private *priv = rdev->priv;
+	int num;
+	unsigned long flags;
+
+	num = rswitch_tx_free(ndev, true);
+	/* We have free descriptors to process new TX packages */
+	if (budget && napi_complete_done(napi, num)) {
```

Comment:
I am not sure if should pass actual number of packets in this function. NAPI documentation says that TX operations can ignore budget.